README.md

```diff
@@ -249,6 +259,57 @@ Expected response:
 }
 ```
 
+#### createDog(dog: <DogInputType>)
+
+Creates a dog in the database for the current user (based on the `google_token` in the params). Returns a DogType object.
+
+Required arguments include:
+* name - String
```

Comment:
Should the required arguments be listed as only `dog:`?  These are the required arguments for the DogInputType.